### PR TITLE
feat: add @capacitor-mlkit/selfie-segmentation package

### PR DESCRIPTION
**Pull request by trancee**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [X] The changes have been tested successfully.
- [ ] A changeset has been created (`npm run changeset`).

I have finally implemented the iOS part. However, I was not able to return an absolute path to the temporary image file that can be displayed in the web app. I am not sure what I am doing wrong, maybe you could have a look at that. It works if I send it as base64 encoded data URI.

### CODE REVIEW

File: packages/selfie-segmentation/ios/Plugin/Classes/ProcessImageResult.swift [hunk 1]
```diff
@@ -1,47 +1,32 @@
...
-        var maskAddress =
-            CVPixelBufferGetBaseAddress(mask.buffer)!.bindMemory(
-                to: Float32.self, capacity: maskBytesPerRow * maskHeight)
-
-        for _ in 0...(maskHeight - 1) {
-            for col in 0...(maskWidth - 1) {
-                // Gets the confidence of the pixel in the mask being in the foreground.
-                let foregroundConfidence: Float32 = maskAddress[col]
-                result.append(foregroundConfidence)
+                result["path"] = url.absoluteString
```
Comment (trancee) [inline packages/selfie-segmentation/ios/Plugin/Classes/ProcessImageResult.swift:21]: @robingenz that's the part I am talking about in the description. The resulting path looks like a valid path to the image file, but when I use it in the web app to show the image, it doesn't work. Do you have an idea what I am doing wrong here?

Comment (robingenz) [reply packages/selfie-segmentation/ios/Plugin/Classes/ProcessImageResult.swift:21]: You plugin code was fine. I just refactored it a bit. It seems like this missing line was the issue: https://github.com/trancee/capacitor-mlkit-plugin-demo/blob/7c0af5afffac9e61f63964f87d425f20e85eb0c2/src/app/modules/selfie-segmentation/selfie-segmentation.page.ts#L55
I've fixed it.

**Review by robingenz — APPROVED** [review]
@trancee LGTM! Can we merge this?

**Comment by trancee** [comment]
> @trancee LGTM! Can we merge this?

Yes, it should be complete now. Thanks for the refactoring!

**Pull request merged**